projects/jsf/src/lib/validator.service.ts

```diff
@@ -84,14 +85,13 @@ export class ValidatorService {
   }
 
   private isUriValid(uri: string): boolean {
-      if (!URI_VALID_CHARS_REGEX.test(uri)) {
+      if (isEmpty(uri) || !URI_VALID_CHARS_REGEX.test(uri)) {
         return false;
       }
 
-      let isValid = isURI(uri);
-      //if invalid, try adding 'https://' (allow scheme to be missing)
```

Comment:
This comment still applies, can you add it back in?

Reply:
Fixed